tasks/c/easy/sum.c

```diff
@@ -2,10 +2,28 @@
 
 #include <stdio.h>
 
-// TODO: Implement the sum function
+int sum(int a, int b){
+    int total  = a + b;
+    return total;
+}
+
+// This is from "https://www.geeksforgeeks.org/how-to-sum-two-integers-without-using-arithmetic-operators-in-cc/"
+// I won't try to explain it as I don't fully understand it.
+int sum_without_plus(int a, int b){
+    char *p = a;
+    return (int)&p[b];
+}
```

Comment:
This Geeks for Geeks example is a bit risky since `char *p = a;` creates a pointer that points straight to the integer value of `a`, but pointers should always point to the memory addresses so it should point to the memory address of `a`. However, this solution does work too in certain compilers but won't necessarily work in every environment.